### ISSUE TITLE
Can't find a link to the Tannin or Mod Organizer page on Patreon. 

### ISSUE THREAD

**Issue by ogrotten** [TanninOne/modorganizer#674]
Just an idea... put up a link to it before the next major release. 

_**I, for one, would use it.**_

Just sayin.

**Comment by TanninOne** [comment]
I appreciate the sentiment but I have a full time job where I can't reduce working hours so even if people did donate, I wouldn't be able to spend more time on MO than I currently am.

**Comment by ogrotten** [comment]
I understand.

However, my intent wasn't to pay you to do more work, it was to donate beer money for work you've already completed. Even if you don't change your schedule, you deserve and have earned it. $10 from me would come without expectation as I've already "received services".

On a related noted, here's anotherway of looking at it: 
What I've done (in a completely different arena) was set a goal where donations matched a single pay period from where I was working at the time. I was then able to take a pay period off from work, unpaid and separate from vacation/pto/sick time, and let the donations pay for my general expenses. I know that workplaces and bosses are all different but I had to work this out significantly ahead of time. 

The amount of work I was able to get done on that project was phenominal with 2 weeks of uninterupted action. 

The bad part was that my sleeping schedule was _gone_ afterwards.

Taking 2 weeks off unpaid notwithstanding, my original point is that I've benefited significantly from your work. 

**And I demand the ability to buy you a beer.**
